Blockzilla/PhotonActionSheet.swift

```diff
@@ -275,14 +282,12 @@ class PhotonActionSheet: UIViewController, UITableViewDelegate, UITableViewDataS
             return
         }
         
-        // Switches can be toggled on/off without dismissing the menu
         if action.accessory == .Switch {
-            let generator = UIImpactFeedbackGenerator(style: .medium)
-            generator.impactOccurred()
```

Comment:
I thought you had mentioned wanting to add haptic feedback. Are you planning on doing it in a different manner?

Reply:
Decided it should be a separate issue to discuss with UX.